Src/library/docking/implementation/controls/sd_horizontal_split_area.e

```diff
@@ -50,6 +50,9 @@ feature {NONE} -- Implementation
 			resize_actions.extend
 				(agent (a_x, a_y, a_width, a_height: INTEGER_32)
 					do remember_top_resize_split_area (Current) end)
+			dpi_changed_actions.extend
+					(agent (a_dpi,a_x, a_y, a_width, a_height: INTEGER_32)
```

Comment:
you should keep same indentation as previous case with resize_actions ...

Reply:
Done